src/bidiMapper/domains/script/ScriptProcessor.ts

```diff
@@ -50,6 +51,21 @@ export class ScriptProcessor {
   async addPreloadScript(
     params: Script.AddPreloadScriptParameters
   ): Promise<Script.AddPreloadScriptResult> {
+    if (params.contexts) {
+      if (params.contexts.length === 0) {
```

Comment:
if zod validates this, we could remove this check

Reply:
Sadly the auto-generated `zod` validator don't validate the one or more in `[+Object]` from CDDL, I will take a peak if its possible to implement. 

CC @jrandolf

Reply:
Can we file a FR here? https://github.com/google/cddlconv

I swear I had filed one already, but I cannot find it.

Reply:
Filed - https://github.com/google/cddlconv/issues/16

Reply:
Thanks